README.md

```
@@ -190,6 +195,18 @@ curl -v -p --proxy-key certs/frontend/private/proxy-client.key --proxy-cert cert
 ### Running on kubernetes
 See following [README.md](examples/kubernetes/README.md)
 
+### Running on a local kubernetes cluster with `kind`
+See this [README.md](examples/kind/README.md) for an example that creates a local kubernetes cluster using` kind` and
```

markdownlint-cli2 (0.22.1)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)